From 64a1c32ee7072d745bb9e95b5b627524fe4ccdf1 Mon Sep 17 00:00:00 2001
From: Robert Schwebel <r.schwebel@pengutronix.de>
Date: Mon, 14 Dec 2009 23:37:35 +0100
Subject: [PATCH] init: revert 'quiet' logic

In 6be918d0ae15ca5f8b3b3879c496205bc8a11c2b the "EXTRA_QUIET" feature
was fixed, but the logic is so that if QUIET is set, it is *not* quiet.

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
---
# 20091214 rsc: submitted here: https://bugs.busybox.net/show_bug.cgi?id=811
 init/init.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/init/init.c b/init/init.c
index fb9c81e..b9c8924 100644
--- a/init/init.c
+++ b/init/init.c
@@ -873,7 +873,7 @@ int init_main(int argc UNUSED_PARAM, char **argv)
 	if (argv[1])
 		xsetenv("RUNLEVEL", argv[1]);
 
-#if ENABLE_FEATURE_EXTRA_QUIET
+#if !ENABLE_FEATURE_EXTRA_QUIET
 	/* Hello world */
 	message(L_CONSOLE | L_LOG, "init started: %s", bb_banner);
 #endif
-- 
1.5.6.5

